### DISCUSSION TITLE
EnchantsSquared Support

### DISCUSSION THREAD

**Discussion by InvisibleSpiders** [QuickShop-Community/QuickShop-Hikari#1593] [Ideas]
Currently, EnchantsSquared enchanted books are only partially supported. 

I would love to see them fully supported. Holograms currently only show up as "Enchanted Book", and players have reported cases of the shop showing "out of stock" while the books are stored in the chest.

**Comment by creatorfromhell** [comment]
> Currently, EnchantsSquared enchanted books are only partially supported.
> 
> I would love to see them fully supported. Holograms currently only show up as "Enchanted Book", and players have reported cases of the shop showing "out of stock" while the books are stored in the chest.

Please test again as we have improved this area dramatically.
